code/game/machinery/medical_kiosk.dm

```diff
@@ -246,7 +246,7 @@
 				var/bit_vol = bit.volume - belly.food_reagents[bit.type]
 				if(bit_vol > 0)
 					chemical_list += list(list("name" = bit.name, "volume" = round(bit_vol, 0.01)))
-	if(altPatient.reagents.addiction_list.len)
+	if(LAZYLEN(altPatient.reagents.addiction_list))
 		for(var/datum/reagent/R in altPatient.reagents.addiction_list)
 			addict_list += list(list("name" = R.name))
```

Comment:
The length check here shouldn't be needed, cause for(var/thing in null) does nothing and doesn't runtime. Or did I miss something in the throng of LAZYness.

Reply:
You're right, this shouldn't be needed. Just being too inflexible about straying too far from the original code.